docs/Decorators.md

```diff
@@ -55,6 +55,8 @@ fastify.get('/', (req, reply) => {
 })
 ```
 
+Note that it is important to keep initial shape of decorated field as close as possible to the value you intend to set dynamically in the future, so instantiate it as a `''` if intended value is string, and as `null` if it will be an object or a function.
+
```

Comment:
I wish I had been given time to review this. I worked hard to eliminate the informal "you" usage from this document. I would write this as:

> Note that it is important to keep initial shape of decorated field as close as possible to the value intended to be set dynamically in the future. Initialize a decorator as a `''` if the intended value is a string, and as `null` if it will be an object or a function.

Reply:
No problem, will make a follow-up PR now.